src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
FWIW, If we do need to support different policies per JIT config point, I can imagine adding a parallel set of APIs to the JIT/EE interface that uses values out of a well-known enumeration as config keys rather than strings. The implementer of the EE side of the interface could then make key-based policy decisions without having to deal with strings.